tests/containers/docker_image.pm

```diff
@@ -28,24 +28,31 @@ use containers::urls 'get_suse_container_urls';
 use version_utils qw(get_os_release check_os_release);
 
 sub run {
-    my ($image_names, $stable_names) = get_suse_container_urls();
     my ($running_version, $sp, $host_distri) = get_os_release;
     my $runtime = "docker";
 
     install_docker_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
-    for my $iname (@{$image_names}) {
-        test_container_image(image => $iname, runtime => $runtime);
-        test_rpm_db_backend(image => $iname, runtime => $runtime);
-        build_container_image(image => $iname, runtime => $runtime);
-        if (check_os_release('suse', 'PRETTY_NAME')) {
-            test_opensuse_based_image(image => $iname, runtime => $runtime);
-            build_with_zypper_docker(image => $iname, runtime => $runtime);
-        }
-        else {
-            exec_on_container($iname, $runtime, 'cat /etc/os-release');
+    # We may test either one specific image VERSION or comma-separated CONTAINER_IMAGES
+    my $versions = (get_var('CONTAINER_IMAGES') =~ /,/) ? get_var('CONTAINER_IMAGES') : get_required_var('VERSION');
```

Comment:
```suggestion
    my $versions = (get_var('CONTAINER_IMAGES') ne "") ? get_var('CONTAINER_IMAGES') : get_required_var('VERSION');
```

This would allow `COUNTER_IMAGES` to be a single image or a list of container images.

Reply:
Should be resoved. Thank you Felix

Reply:
Can you try without `ne ""` ? The condition should return `false` if the variable is not set and then it will pick the second statement.

Reply:
ok this latest version should really be replaced with what I suggested before : 
`$versions = get_var('CONTAINER_VERSIONS', get_required_var('VERSION')) ;`

Reply:
yes, that also works and it's nicer